.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: c
+dist: trusty   # Travis build machine is Ubuntu Trusty 14.04 x64
+sudo: required # A sudo enabled, full VM is used for building instead of a docker container
+
+git:
+  submodules: false
+
+before_install:
+  - sudo apt-get update -qq
+  # Pre-reqs for building omsagent
+  - sudo apt-get install -qq git pkg-config make g++ rpm librpm-dev libpam0g-dev libssl-dev libkrb5-dev curl libcurl4-gnutls-dev  
+
+# We are in /home/travis/build/Microsoft/Powershell-DSC-for-Linux/
+
+
+install: 
+  # Clone OMI
+  - cd ..
```

Comment:
I debated whether do do that.  It does take longer, but I agree that it is more complete.
I'll update this commit.  ;)